src/components/ToDoList.js

```diff
@@ -0,0 +1,21 @@
+import "./ToDo.css";
+import ToDo from "./ToDo.js";
+
+function ToDoList({ toDo, onTogglePending, onDeleteToDo }) {
```

Comment:
I'd change the prop `toDo` to be called `toDos`.
Remember that if you change it here, you need to change it also where you are using the `ToDoList` component.